src/query-builder/update-result.ts

```diff
@@ -1,15 +1,24 @@
+import { setEnumerableProperties } from '../util/object-utils.js'
+
 export class UpdateResult {
   readonly #numUpdatedRows: bigint
+  readonly #numChangedRows: bigint
 
-  constructor(numUpdatedRows: bigint) {
+  constructor(numUpdatedRows: bigint, numChangedRows: bigint) {
     this.#numUpdatedRows = numUpdatedRows
+    this.#numChangedRows = numChangedRows
   }
 
   get numUpdatedRows(): bigint {
     return this.#numUpdatedRows
   }
+
+  /**
+   * MySQL-only
```

Comment:
Instead of documenting this as `MySQL-only` document it as optional and that some dialects may not provide a value. Some 3rd party dialects might have this and then the documentation would be incorrect for those ones.